packages/fdc3-agent-proxy/src/channels/DefaultChannel.ts

```diff
@@ -24,47 +23,57 @@ export class DefaultChannel implements Channel {
   }
 
   async broadcast(context: Context): Promise<void> {
-    const request: BroadcastRequest = {
-      meta: this.messaging.createMeta(),
-      payload: {
-        channelId: this.id,
-        context,
-      },
-      type: 'broadcastRequest',
-    };
-    await this.messaging.exchange<BroadcastResponse>(request, 'broadcastResponse');
+    const done = await this.messaging.exchange<BroadcastResponse>(
+      {
+        meta: this.messaging.createMeta(),
+        payload: {
+          channelId: this.id,
+          context,
+        },
+        type: 'broadcastRequest',
+      } as BroadcastRequest,
+      'broadcastResponse'
+    );
+    console.log('broadcast done', done);
```

Comment:
If this `console.log` is supposed to be here, please use `console.debug`.

Reply:
again it came back with that bad merge 🤦